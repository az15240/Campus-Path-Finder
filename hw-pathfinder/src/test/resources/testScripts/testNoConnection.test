# Tests the Dijkstra function on two disconnected nodes in a graph

CreateGraph g

AddNode g A
AddNode g B
AddNode g C

AddEdge g A B 1
AddEdge g B A 2
AddEdge g A A 3

ListNodes g

ListChildren g A

FindPath g A C
FindPath g C A