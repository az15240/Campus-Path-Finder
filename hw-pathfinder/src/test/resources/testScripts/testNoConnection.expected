# Tests the Dijkstra function on two disconnected nodes in a graph

created graph g

added node A to g
added node B to g
added node C to g

added edge 1.000 from A to B in g
added edge 2.000 from B to A in g
added edge 3.000 from A to A in g

g contains: A B C

the children of A in g are: A(3.000) B(1.000)

path from A to C:
no path found
path from C to A:
no path found